microshift_running_apps/microshift-operators-olm.adoc

```diff
@@ -0,0 +1,71 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-operators-olm"]
+= Using Operator Lifecycle Manager with {microshift-short}
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-operators-olm
+
+toc::[]
+
+OLM is used in {microshift-short} for installing and running optional link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/architecture/control-plane#olm-operators_control-plane[add-on Operators].
+
+* Cluster Operators as applied in {ocp} are not used in {microshift-short}.
+* You must add your own catalogs for the Operators you want to use with your applications.
+* You must use the CLI to conduct OLM activities with {microshift-short}. The console and OperatorHub GUIs are not available.
+* Install the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/cli_tools/opm-cli#cli-opm-install[`opm` CLI]
+
+[IMPORTANT]
+====
+Before using an Operator, verify with the provider whether the Operator is supported on {product-title}.
+====
+
+[id="microshift-installing-olm-options_{context}"]
+== Determining your installation type for OLM
+There are different ways to install OLM with {microshift-short}, depending on your use case.
+
+* To use OLM with {microshift-short}, you can install the OLM RPM along with the {microshift-short} RPM on {op-system-base-full}.
+//future xref to come once install doc is complete
+* You can also embed OLM in a {op-system-ostree-first} image. When embedding in a {op-system-ostree} image, use the installation path that matches your intended use case: either with a network connection, or without one.
+//** See future xref to come once install doc is complete
+//** See for a disconnected installation (xref to come once doc complete)
+
+[id="microshift-options-adding-operator-catalogs_{context}"]
+== Options for adding custom Operator catalogs
+You can add custom Operator catalogs that you can manage with OLM in one of two ways:
+
+* If you know the exact few Operators you want to use, you can use file-based Operator catalogs, see the example catalog in the "File-based Operator catalogs" section.
+* If you want to choose from a list of Operators that are supported with {microshift-short}, you can use the procedure for disconnected installations using a mirror registry.
```

Comment:
But they still need a catalog. MicroShift doesn't have it's own catalog like openshift does, so they will need to create the catalog or mirror openshift's one.
Is that the meaning here?

I think connected/disconnected and crafted/mirrored catalog are two separate things and can be mixed together in any way.

Reply:
> If you want to choose from a list of Operators that are supported with MicroShift, you can use the procedure for disconnected installations using a mirror registry.

Similarly, here, I'd think it's better to call out the "filtering an existing catalog image".